internal/config/helpers.go

```diff
@@ -34,6 +34,15 @@ func InProdClowder() bool {
 	return clowder.IsClowderEnabled() && strings.HasPrefix(*clowder.LoadedConfig.Metadata.EnvName, "env-stage")
 }
 
+// RequestedTopic returns mapped topic from Clowder. When not running in Clowder mode, it returns the input topic name.
+func RequestedTopic(topic string) string {
+	if t, ok := topicRequest[topic]; ok {
+		return t
```

Comment:
do we need our mapping here? I think this would work:
```suggestion
    if InClowder() {
	    if t, ok := cfg.Kafka.Topics[topic]; ok {
	        return t.Name
	    }
```

Reply:
Are you sure? `cfg.Kafka.Topics` is a slice, not map. We could search the slice everytime but we will be sending many messages so I wanted a map so it will scale.

Reply:
Turns out you mean https://github.com/RedHatInsights/app-common-go/blob/811f8bb0662990c714e87d62f633940f05390de0/pkg/api/v1/config.go#L13 which is indeed a map, Clowder does exactly what I do in our codebase, let's use it.

Reply:
You're rigth the correct thing to use would be `cfg.KafkaTopics`